ceph/rados/rados_service.py

```diff
@@ -0,0 +1,105 @@
+"""
+   This module contains the methods to perform general ceph cluster modification operations.
+    1. Remove OSD
+    2. Add OSD
+    3. Set osd out
+    3. Zap device path
+"""
+import logging
+
+from ceph.rados.core_workflows import RadosOrchestrator
+
+log = logging.getLogger(__name__)
+
+
+class RadosService(RadosOrchestrator):
+    """
+    Perform general cluster modification operations
+    """
+
+    def run_command(self, cmd: str):
+        """
+        Runs ceph commands  for the action specified
+        Args:
+            cmd: Command that needs to be run
+        Returns:
+            out (Str), err (Str) stdout and stderr response
+        """
+        try:
+            out, err = self.node.shell([cmd])
+        except Exception as er:
+            log.error(f"Exception hit while command execution. {er}")
+            return None
+        return out, err
```

Comment:
This method looks similar to what we have in the parent object `run_ceph_command`. Do we need another one?

Reply:
In ` run_ceph_command ` we only return the output, and that too with format changed to dictionary. 

In this case, we would need the err stream too, without any further formatting. I feel we can keep this method. 
@psathyan , if you feel it's not required, we can make the ` node.shell() ` calls directly. 

@tintumathew10 , let me know your thoughts on this.